xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -684,6 +712,32 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 
       // associate with current window
       [appWindow setContentView:view];
+
+      // set the window to the appropriate screen and screen position
+      if (m_bFullScreen)
+      {
+        [appWindow setFrameOrigin:screen.frame.origin];
+      }
+      else
+      {
+        // if there are stored window positions use that as the origin point
+        const int top = settings->GetInt(SETTING_WINDOW_TOP.UTF8String);
+        const int left = settings->GetInt(SETTING_WINDOW_LEFT.UTF8String);
+
+        NSPoint windowPos;
+        if (top != 0 && left != 0)
```

Comment:
If you "stick" window to the left edge, won't left == 0? I think you should use || instead of &&

Reply:
right, if any of the stored points is not the default we should use the stored position